mavros_extras/src/plugins/vision_speed_estimate.cpp

```diff
@@ -46,6 +48,7 @@ class VisionSpeedEstimatePlugin : public plugin::PluginBase {
 
 		if (listen_twist)
 			vision_vel_sub = sp_nh.subscribe("speed_twist", 10, &VisionSpeedEstimatePlugin::vel_twist_cb, this);
+
```

Comment:
Should not be there.

Reply:
actually I did this PR quickly and didn't realize I forgot to add the `speed_twist_cov` option.